numpy/random/_generator.pyx

```diff
@@ -389,6 +389,23 @@ cdef class Generator:
         .. [3] Wikipedia, "Exponential distribution",
                https://en.wikipedia.org/wiki/Exponential_distribution
 
+        Examples
+        --------
+        A real world example: Assume a company has 10000 customer support 
+        agents and each agent spends on average 4 minutes with a 
+        customer.
+
+        >>> n = 10000
+        >>> z = np.random.default_rng().exponential(scale=4, size=n)
+
+        What is the probability an agent will spend between 4
+        to 5 minutes with a customer? 
+
+        >>> x = ((d < 5).sum())/n 
+        >>> y = ((d < 4).sum())/n
+        >>> x-y
+        0.08 # may vary
```

Comment:
```suggestion
        >>> x-y # will vary
```

Reply:
Thank you! I've made the edit

Reply:
yes, sorry, I think this was my mistake on the call.